go/api/bitmap_commands.go

```diff
@@ -0,0 +1,99 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+import "github.com/valkey-io/valkey-glide/go/glide/api/options"
+
+// Supports commands and transactions for the "Bitmap" group of commands for standalone and cluster clients.
+//
+// See [valkey.io] for details.
+//
+// [valkey.io]: https://valkey.io/commands/#bitmap
+type BitmapCommands interface {
+	// Sets or clears the bit at offset in the string value stored at key.
+	// The offset is a zero-based index, with `0` being the first element of
+	// the list, `1` being the next element, and so on. The offset must be
+	// less than `2^32` and greater than or equal to `0` If a key is
+	// non-existent then the bit at offset is set to value and the preceding
+	// bits are set to 0.
+	//
+	// Parameters:
+	//  key - The key of the string.
+	//  offset - The index of the bit to be set.
+	//  value - The bit value to set at offset The value must be 0 or 1.
+	//
+	// Return value:
+	//  The bit value that was previously stored at offset.
+	//
+	// Example:
+	//  result, err := client.SetBit("key", 1, 1)
+	//  result.Value(): 1
+	//  result.IsNil(): false
+	//
+	// [valkey.io]: https://valkey.io/commands/setbit/
+	SetBit(key string, offset int64, value int64) (Result[int64], error)
```

Comment:
Never returns `null`/`nil` - please fix the impl and example
```suggestion
	SetBit(key string, offset int64, value int64) (int64, error)
```

Reply:
done